hubblestack/extmods/fileserver/roots.py

```diff
@@ -32,7 +32,6 @@
 import hubblestack.utils.platform
 import hubblestack.utils.stringutils
 import salt.utils.versions
```

Comment:
Same for this, should be under hubblestack.utils.version

Reply:
Is it version or versions? I changed it to versions to match the salt module naming schema.